client/src/components/CarInfoSection/index.tsx

```diff
@@ -0,0 +1,155 @@
+            <CalendarTodayOutlinedIcon />
+            <Typography className="info-value" variant="body1">{carInfo.year}</Typography>
+            <span className="dot-divider">.</span>
+            <SpeedTwoToneIcon />
+            <Typography className="info-value" variant="body1">
+              {`${carInfo.mileage} KM`}
+            </Typography>
+            <span className="dot-divider">.</span>
+            <Box sx={{ position: 'relative', display: 'inline-flex' }}>
+              <CircularProgress
+                sx={{ marginTop: '-5px', color: '#393d47' }}
+                variant="determinate"
+                value={carInfo.quality}
+              />
+              <Box
+                sx={{
+                  top: -1,
+                  left: 4,
+                  bottom: 0,
+                  right: 0,
+                  position: 'absolute',
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
+              >
+                <Typography
+                  variant="caption"
+                  color="text.secondary"
+                >
+                  {carInfo.quality}
+                  %
+                </Typography>
+              </Box>
+            </Box>
+
+          </div>
+          <div className="info-container">
+            <LocationOnIcon />
+            <Typography className="info-value" variant="body1">{carInfo.location}</Typography>
+            <span className="dot-divider">.</span>
+            <LocalGasStationIcon />
+            <Typography className="info-value" variant="body1">{carInfo.fuel}</Typography>
+          </div>
+
+        </section>
+
+        <Typography className="subtitle" variant="subtitle1"> Car overview</Typography>
+        <section className="main-info-container overview">
+          {carInfo.transmission
+            ? (
+              <Paper className="feature-card" elevation={2}>
+                <Box
+                  className="feature-image"
+                  component="img"
+                  alt={`transmission ${carInfo.transmission}`}
+                  src={features[carInfo.transmission === 'auto' ? 'Automatic Transmission' : 'Manual Transmission']}
+                />
+              </Paper>
+            ) : ''}
+
+          {carInfo.features.map((feature) => (
+            <Paper key={feature} className="feature-card" elevation={2}>
+              <Box
+                className="feature-image"
+                component="img"
+                alt={feature}
+                src={features[feature]}
+              />
+            </Paper>
+          ))}
+        </section>
+
+        <Typography className="subtitle" variant="subtitle1">Description</Typography>
+        <section className="main-info-container description">
+
+          <Typography variant="body1">
+            {carInfo.description}
+            .
+          </Typography>
+        </section>
```

Comment:
there's a possibility that thesr's no description for specific card, try to handle this section conditionally